Algorithms/src/main/java/dynamic_programing/Factorial.java

```diff
@@ -0,0 +1,5 @@
+Public class Factorial{
+    public int factorial(int n){
```

Comment:
Since this is not a dp solution can you please find some other directory to put this?